installing/installing_vsphere/installing-restricted-networks-vsphere.adoc

```diff
@@ -41,7 +41,12 @@ include::modules/installation-about-restricted-network.adoc[leveloffset=+1]
 
 include::modules/cluster-entitlements.adoc[leveloffset=+1]
 
-include::modules/installation-vsphere-infrastructure.adoc[leveloffset=+1]
+include::modules/
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about CSI automatic migration, see "Overview" in xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere[VMware vSphere CSI Driver Operator]..adoc[leveloffset=+1]
```

Comment:
```suggestion
* For more information about CSI automatic migration, see "Overview" in xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere[VMware vSphere CSI Driver Operator]
```

Reply:
Yeah, thanks for spotting this. An awful copy and paste for sure. Fixed!